src/watcher.js

```diff
@@ -216,6 +217,18 @@ class Watcher extends SafeEventEmitter {
         this.on('load.chat', () => observe(chatWatcher, $('.chat__container')[0]));
     }
 
+    darkObserver() {
```

Comment:
you can just do something similar to `chat_state/index` instead of putting it here.. this file is mainly for watching dom changes.